dsp/src/iir_int.rs

```diff
@@ -1,6 +1,21 @@
 use serde::{Deserialize, Serialize};
 
-pub type IIRState = [i32; 5];
+#[derive(Copy, Clone, Default, Deserialize, Serialize)]
+pub struct IIRState(pub [i32; 5]);
+
+impl IIRState {
+    #[inline(always)]
+    pub fn get_x(&self, index: usize) -> i32 {
+        // x0 is at index 0 in a biquad between updates
```

Comment:
Should we add protection to prevent the user from accidentally overreaching the bounds of x?